spec/lib/workers/jobs/jobs_spec.rb

```diff
@@ -24,4 +24,20 @@
       )
     end
   end
+
+  describe "#miq_schedule_queue_scheduled_work" do
+    require "rufus-scheduler"
+
+    it "delegates the queueing to MiqSchedule" do
+      miq_schedule = stub_const("MiqSchedule", double("MiqSchedule"))
```

Comment:
good point! fixed in https://github.com/ManageIQ/manageiq/commit/7fe966209a27dfcc1ed0b040826d4292b4e343f6